ern-container-gen-ios/src/hull/ElectrodeContainer/ElectrodeReactNative.h

```diff
@@ -101,23 +105,42 @@ NS_ASSUME_NONNULL_BEGIN
                            properties:(NSDictionary * _Nullable)properties;
 
 /**
- Returns a react native miniapp (from a JSBundle) inside a view controller.
+ Returns a react native miniapp (from a JSBundle).
 
  @param name The name of the mini app, preferably the same name as the jsbundle
  without the extension.
  @param properties Any configuration to set up the mini app with.
- @return A UIViewController containing the view of the miniapp.
+ @return a UIView of the miniapp.
  */
 - (UIView *)miniAppViewWithName:(NSString *)name
                      properties:(NSDictionary *_Nullable)properties;
 
 /**
+ Returns a react native miniapp (from a JSBundle).
+
+ @param name The name of the mini app, that is registered with the AppComponent.
+ @param properties initialprops for a React Native miniapp.
+ @param sizeFlexibilty defines size flexibility type of the root view
+ @return a UIView of the miniapp.
+ */
+- (UIView *)miniAppViewWithName:(NSString *)name
+                     properties:(NSDictionary *_Nullable)properties
+                sizeFlexibility:(NSInteger)sizeFlexibilty
+        __attribute((deprecated("use -miniAppViewWithName:properties:sizeFlexibility:delegate instead")));
+
+/**
+ Returns a react native miniapp (from a JSBundle).
+
  @param name The name of the mini app, that is registered with the AppComponent.
  @param properties initialprops for a React Native miniapp.
  @param sizeFlexibilty defines size flexibility type of the root view
- @return UIView
+ @param delegate of the root view that handles intrinsic size updates
```

Comment:
oh BTW, could you remove this comment?

Reply:
@qw1024 - While you fix this up, please also remove `PR comments` from the commit message body.

Reply:
done